config/locales/en/providers.yml

```diff
@@ -416,6 +416,7 @@ en:
           unknown_frequency: Frequency unknown
           residual_balance: Capital Contribution - Residual Balance Check
           policy_disregards: Discretionary disregards selected
+          multi_benefit: Multiple benefits in single bank transaction
```

Comment:
Is this needed? It doesn't seem to appear in the other files you've changed.

Reply:
Ran into an error on localhost using Mock True Layer - required a translation so it's just covering that. Will tag you in the thread on apply dev :)

Reply:
Ah, I knew there would be a good reason 😄